stewards.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "ca": "standard_crypto/server_ca.crt",
-        "endpoint": "https://steward.sommelier.standardcryptovc.com:5734"
+        "ca": "teku/server_ca.crt",
+        "endpoint": "https://sommelier.teku.network:5734"
     }
```

Comment:
Please append your information as a new JSON item instead of overwriting Standard Crypto's